akka-docs/rst/experimental/index-java.rst

```diff
@@ -19,10 +19,10 @@ prior deprecation.
 .. toctree::
    :maxdepth: 1
 
-   ../java/persistence
    ../dev/multi-node-testing
    ../java/lambda-actors
    ../java/lambda-fsm
+   ../java/persistence-query
```

Comment:
Whoops, so I've put it in the wrong section: https://github.com/akka/akka/blame/master/akka-docs/rst/java/index-actors.rst#L15
Would you remove it from there in this PR as well?